lib/hca/settings.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+module HCA
+  module Settings
+    CONFIG = Rails.application.config_for(:health_care_application).freeze
+
+    def self.cert_store(paths)
+      store = OpenSSL::X509::Store.new
+      Array(paths).each do |path|
+        store.add_file(Rails.root.join('config', 'health_care_application', 'certs', path).to_s)
```

Comment:
RE: this and storing them in devops, since these are required to connect to the Dev or QA systems for testing, I think we may need to keep both the certs and this store code around for local development work.